app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split()
+    if len(command_list) == 3 and command_list[0] == "mv":
+        old_name, new_name = command_list[1:]
+        if "/" not in new_name:
+            pass
+        else:
+            new_name_split = new_name.split("/")
+            path = new_name_split[0]
+            for index in range(1, len(new_name_split)):
+                if os.path.exists(path) is False:
+                    os.makedirs(path)
+                    path += f"/{new_name_split[index]}"
```

Comment:
The logic for building the path and checking its existence is flawed. The path should be updated before checking its existence. Consider updating the path before the `os.path.exists(path)` check, and ensure the path is correctly constructed for each iteration.